qwandaq/src/main/java/life/genny/qwandaq/utils/QwandaUtils.java

```diff
@@ -544,36 +544,66 @@ public Ask generateAskGroupUsingBaseEntity(BaseEntity baseEntity) {
 		entityMessage.setReplace(true);
 
 		// create a child ask for every valid atribute
-		baseEntity.getBaseEntityAttributes().stream()
-				.filter(ea -> defBE.containsEntityAttribute("ATT_" + ea.getAttributeCode()))
+		defBE.getBaseEntityAttributes().stream()
+//		baseEntity.getBaseEntityAttributes().stream()
+//				.filter(ea -> defBE.containsEntityAttribute("ATT_" + ea.getAttributeCode()))
 				.forEach((ea) -> {
+					String attributeCode = StringUtils.removeStart(ea.getAttributeCode(),"ATT_");
```

Comment:
This can be abstracted out into defUtils using some kind of static list of known def attribute prefixes

Reply:
declare prefix constants in DefUtils to be used in other classes